site/styles/_variables.scss

```diff
@@ -36,76 +89,83 @@ $spacer: math.div(5, 8) * 1rem; // 10px
 
 $spacers: (
   '0': 0,
-  'p5':  ($spacer * 0.5),  // 5px
-  'p75': ($spacer * 0.75), // 7.5px
-  '1':   $spacer,          // 10px
-  '2':   ($spacer * 2),    // 20px
-  '2p5': ($spacer * 2.5),  // 25px
-  '3':   ($spacer * 3),    // 30px
-  '4':   ($spacer * 4),    // 40px
-  '5':   ($spacer * 5),    // 50px
+  'p5': (
+    $spacer * 0.5,
+  ),
+  // 5px
+  'p75':
+    (
+      $spacer * 0.75,
+    ),
+  // 7.5px
+  '1': $spacer,
+  // 10px
+  '2':
+    (
+      $spacer * 2,
+    ),
+  // 20px
+  '2p5':
+    (
+      $spacer * 2.5,
+    ),
+  // 25px
+  '3':
+    (
+      $spacer * 3,
+    ),
+  // 30px
+  '4':
+    (
+      $spacer * 4,
+    ),
+  // 40px
+  '5':
+    (
+      $spacer * 5,
+    ),
+  // 50px
```

Comment:
This looks like auto-formatting making it harder to read...

Reply:
Yeah I'll try to figure out how to fix that.